subsys/bluetooth/mesh/brg_cfg_cli.c

```diff
@@ -5,12 +5,158 @@
  */
 
 #include <zephyr/bluetooth/mesh.h>
+#include "access.h"
+#include "foundation.h"
+#include "msg.h"
 
 #define LOG_LEVEL CONFIG_BT_MESH_MODEL_LOG_LEVEL
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_mesh_brg_cfg_cli);
 
+static int32_t msg_timeout;
+
+static struct bt_mesh_brg_cfg_cli *cli;
+
+static int subnet_bridge_status(const struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
+				struct net_buf_simple *buf)
+{
+	enum bt_mesh_subnet_bridge_state status =
+		(enum bt_mesh_subnet_bridge_state)net_buf_simple_pull_u8(buf);
+	enum bt_mesh_subnet_bridge_state *rsp;
+
+	if (bt_mesh_msg_ack_ctx_match(&cli->ack_ctx, OP_SUBNET_BRIDGE_STATUS, ctx->addr,
+				      (void **)&rsp)) {
+		*rsp = status;
+		bt_mesh_msg_ack_ctx_rx(&cli->ack_ctx);
+	}
+
+	if (cli->cb && cli->cb->subnet_bridge_status) {
+		cli->cb->subnet_bridge_status(cli, ctx->addr, status);
+	}
+	return 0;
+}
+
+static int bridging_table_status(const struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
+				struct net_buf_simple *buf)
+{
+	struct bt_mesh_bridging_table_status table_status;
+	struct bt_mesh_bridging_table_status *rsp;
+	uint32_t net_key_indices;
+
+	table_status.status = (enum bt_mesh_subnet_bridge_state)net_buf_simple_pull_u8(buf);
```

Comment:
This is quite arguable. I guess we need to discuss it in IRL before doing such changes. I suggest leave it `as is`. Since this is a collaboration branch, we can change it later.